### PR TITLE
Better upload screen

### PR DESCRIPTION
**Pull request by jonbell-lot23**
* Screen defaults to the save button, so you can just press enter
* Better focus state for the discard link
* Better focus state for the edit section

I'm making this a draft so I can discuss the details with Ollie tomorrow.

### CODE REVIEW

File: src/ui/components/UploadScreen/UploadScreen.tsx [hunk 1]
```diff
@@ -56,29 +56,35 @@ function Actions({ onDiscard, status }: { onDiscard: () => void; status: Status
   const isDeleting = status === "deleting";
   const shouldDisableActions = isSaving || isDeleting;
 
+  useEffect(() => {      
+    document.getElementById("saveButton").focus(); // for easier upload
```
Comment (jasonLaster) [inline src/ui/components/UploadScreen/UploadScreen.tsx:60]: You can use a ref for this

https://reactjs.org/docs/refs-and-the-dom.html#creating-refs

`saveButtonRef.focus()`

**Comment by jonbell-lot23** [comment]
Using refs instead.

cc @oliverdealb -- when this goes live, we can share it out

**Review by jasonLaster — APPROVED** [review]

File: src/ui/components/UploadScreen/UploadScreen.tsx [hunk 2]
```diff
@@ -55,30 +55,36 @@ function Actions({ onDiscard, status }: { onDiscard: () => void; status: Status
   const isSaving = status === "saving";
   const isDeleting = status === "deleting";
   const shouldDisableActions = isSaving || isDeleting;
+  const saveButtonRef = React.useRef(null);
+
+  useEffect(() => {      
+    saveButtonRef.current.focus(); // for faster/more keyboard accessible uploads
```
[the remark this answers is not in the document]
Comment (jonbell-lot23) [reply src/ui/components/UploadScreen/UploadScreen.tsx:61]: Thanks @bvaughn ! I didn't know about either of those commands, are they in our docs somewhere?

[the remark this answers is not in the document]
Comment (bvaughn) [reply src/ui/components/UploadScreen/UploadScreen.tsx:61]: I don't know. They're just tasks that CI runs. You can run them locally to save yourself the surprise of CI failing :)

**Pull request merged**